src/main/java/seedu/duke/commands/AddCommand.java

```diff
@@ -29,7 +30,7 @@ public AddCommand(String[] input) throws NullPointerException {
         }
         //check that the episodes do not exceed seasons
 
-        Show show = new Show(input[1], Integer.parseInt(input[2]), seasonEpisodes);
+        Show show = new Show(input[1], Integer.parseInt(input[2]), Integer.parseInt(input[3]), seasonEpisodes);
```

Comment:
Since the variables are already declared, why not pass use them? Like ShowDuration instead of Integer.parseInt(2)

Reply:
Good catch, whoever is in charge of the add command just adjust accordingly.